README.md

```diff
@@ -35,16 +26,16 @@ input.
 #### Metadata
 
 In general, since fw-classification acts on input metadata, the input file needs to have
-it's metadata populated before running file-classifier. The metadata can live in a few
+it's metadata populated before running form-classifier. The metadata can live in a few
 places depending on how the file will be classified.  The most common would be in the
-`file.info.header.<file-type>` which will be populated by `file-metadata-importer`.  But
+`file.info.header.<file-type>` which will be populated by `form-importer`.  But
 the metadata can also be in a separate file such as the `sidecar.json` for NIfTIs, or in
```

Comment:
Should remove reference to Nifti.  Also, I'm not certain this can actually save json sidecars...

Reply:
Fixed